discovery-infra/download_logs.py

```diff
@@ -224,6 +236,25 @@ def download_must_gather(kubeconfig: str, dest_dir: str):
         log.warning(f"Failed to run must gather: {ex}")
 
 
+def gather_sosreport_data(output_dir: str):
+    sosreport_output = os.path.join(output_dir, "sosreport")
+    recreate_folder(sosreport_output)
+    controller = LibvirtController(private_ssh_key_path=private_ssh_key_path_default)
+    for node in controller.list_nodes():
+        try:
+            gather_sosreport_from_node(node, sosreport_output)
+        except (TimeoutError, IndexError, SSHException) as e:
```

Comment:
why not to catch any exception too? i think it is better not to fail and bring all we can then to write nice exception code

Reply:
if we have a real error?